core/trino-main/src/main/java/io/trino/sql/planner/iterative/rule/RemoveRedundantIdentityProjections.java

```diff
@@ -33,7 +33,7 @@
             // of its child
             .matching(RemoveRedundantIdentityProjections::outputsSameAsSource);
 
-    private static boolean outputsSameAsSource(ProjectNode node)
+    public static boolean outputsSameAsSource(ProjectNode node)
```

Comment:
Instead of making it public, I think it's better to have this method inside `FuseCrossJoinedGlobalAggregations`

Reply:
then I would have to copy it to `PlanNodeFuser` as well.

Reply:
I think it's fine to have it inside `PlanNodeFuser` which can be used by `FuseCrossJoinedGlobalAggregations`. It's just that it might not be a good idea to depend on other rule public methods.

Reply:
Ok, i copied it to the `PlanNodeFuser`